desktop/components/current_auctions/README.md

```diff
@@ -0,0 +1,60 @@
+# Current Auctions
+
+A React component that displays a bar of four current Auctions.
+
+## Usage
+
+If using within a React context:
+
+```javascript
+import { CurrentAuctions } from 'components/current_auctions'
+
+class Page extends Component {
+  render() {
+    return (
+      <CurrentAuctions
+        sales={sales}
+      />
+    )
+  }
+}
+```
+
+If used within a non-React context, `index.jsx` will automatically mount provided there is a default `#react-mount-current-auctions` somewhere present in the DOM:
```

Comment:
> Then it wouldn't necessarily need documentation of the implicit behavior.

This is a good point. I guess the idea I had was to suggest a usage pattern based on a default (an `#id`, the keyword `react-mount` to indicate that it breaks overall app convention, and then the name of the component being mounted, `current-auctions`). For most cases a user of the component would scan the `README` and implement exactly, without thinking too much about it. That being said I can totally see how creating that contract in the docs is essentially the same thing, and how it could possibly cross some barriers.  Saying "Pass in some data and a mount point" is likely more than enough.